symphony/bdk/core/service/message/message_service.py

```diff
@@ -34,6 +35,38 @@ def __init__(self, messages_api: MultiAttachmentsMessagesApi, message_suppressio
         self._auth_session = auth_session
         self._retry_config = retry_config
 
+    async def send_simple_message(
+            self,
+            stream_id: str,
+            message: str
+    ) -> V4Message:
+        """Send a message to an existing stream.
+        See: `Create Message <https://developers.symphony.com/restapi/reference#create-message-v4>`_
+
+        :param stream_id: The ID of the stream to send the message to.
+        :param message: The MessageML content to be sent. If <messageML> tags are not put,
+          they will be added before sending the message.
+
+        :return: a V4Message object containing the details of the sent message.
+        """
+        return await self.send_complex_message(stream_id, Message(content=message))
+
+    async def send_complex_message(
+            self,
+            stream_id: str,
+            message: Message
+    ) -> V4Message:
+        """Send a message to an existing stream.
+        See: `Create Message <https://developers.symphony.com/restapi/reference#create-message-v4>`_
+
+        :param stream_id: The ID of the stream to send the message to.
+        :param message: the message to be sent
+
+        :return: a V4Message object containing the details of the sent message.
+        """
+        return await self.send_message(stream_id, message.content, message.data, message.version, message.attachments,
+                                       message.previews)
+
     @retry
     async def send_message(
```

Comment:
don't we want data to be a dict/object instead of a string?
and could we make attachment/preview a list of tuples? [(att1, prev1),...] ?